### PR TITLE
Re-add bootstrap.properties file to maintain downstream processes and increase accounts.maxNumber=20_000_000

### LINKED ISSUE #8914: Re-add bootstrap.properties file for downstream sanity
In a [previous PR](https://github.com/hashgraph/hedera-services/pull/8667), the `bootstrap.properties` file for mainnet was removed. However, it's causing problems creating the artifact downstream. We need to re-add this file with a comment so that it doesn't get deleted.

### PR DESCRIPTION
**Pull request by mhess-swl**
Fixes #8914 

EDIT: we're also upping the number of max accounts to 20 million

### CODE REVIEW

**Review by ElijahLynn — APPROVED** [review]
This is great Matt thanks for doing this until we figure out a more betterer long term solution!

File: hedera-node/configuration/mainnet/bootstrap.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# This file is empty, but needs to remain in the repo in order to not
+# break downstream artifact creation scripts.
```
[the remark this answers is not in the document]
Comment (mhess-swl) [reply hedera-node/configuration/mainnet/bootstrap.properties]: I reworded this. Give it a read and see what you think.

[the remark this answers is not in the document]
Comment (ElijahLynn) [reply hedera-node/configuration/mainnet/bootstrap.properties]: Thanks, that looks more comprehensive now.

File: hedera-node/configuration/mainnet/bootstrap.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# This file needs to exist whether or not there are any properties in it.
+# Downstream processes depend on its existence.
```
Comment (tannerjfco) [inline hedera-node/configuration/mainnet/bootstrap.properties:2]: nit, no empty line at end of file

Comment (mhess-swl) [reply hedera-node/configuration/mainnet/bootstrap.properties]: fixed

File: hedera-node/configuration/mainnet/bootstrap.properties [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# This file needs to exist whether or not there are any properties in it.
```
Comment (poulok) [inline hedera-node/configuration/mainnet/bootstrap.properties:1]:
```suggestion
# This file needs to exist even though there are any properties in it. Do NOT put properties in this file.
```

Comment (ElijahLynn) [reply hedera-node/configuration/mainnet/bootstrap.properties:1]: We need a "why" in this. Somehow it got removed.

Comment (ElijahLynn) [reply hedera-node/configuration/mainnet/bootstrap.properties:1]: Nvm, I needed to scroll down. It is there.

**Review by poulok — APPROVED** [review]

**Review by ElijahLynn — APPROVED** [review]
LGTM! 🚀

**Review by Neeharika-Sompalli — APPROVED** [review]

**Review by nathanklick — APPROVED** [review]

**Pull request merged**